iguana.corecontroller/src/main/java/org/aksw/iguana/cc/worker/impl/SPARQLWorker.java

```diff
@@ -14,8 +15,8 @@
 @Shorthand("SPARQLWorker")
 public class SPARQLWorker extends HttpGetWorker {
 
-	public SPARQLWorker(String taskID, Connection connection, String queriesFile, @Nullable String responseType, @Nullable String parameterName, @Nullable Integer timeOut, @Nullable Integer timeLimit, @Nullable Integer fixedLatency, @Nullable Integer gaussianLatency, Integer workerID) {
-		super(taskID, connection, queriesFile, responseType, parameterName, "lang.SPARQL", timeOut, timeLimit, fixedLatency, gaussianLatency, workerID);
-	}
+    public SPARQLWorker(String taskID, Integer workerID, Connection connection, Map<String, Object> queries, @Nullable Integer timeLimit, @Nullable Integer timeOut, @Nullable Integer fixedLatency, @Nullable Integer gaussianLatency, @Nullable String parameterName, @Nullable String responseType) {
```

Comment:
Is there any value in keeping SPARQLWorker ATM beyond backwards compatibility? Don't do HttpGetWorker (and HttpPostWorker) the job?